.github/workflows/flowzone.yml

```diff
@@ -301,33 +301,37 @@ jobs:
       versionbot: ${{ steps.versionbot_checkout.outcome }}
 
     steps:
-      - name: Checkout versioned source
-        id: versionbot_checkout
-        continue-on-error: true
-        uses: actions/checkout@v3
-        with:
-          ref: versionbot/pr/${{ github.event.pull_request.number }}
-          fetch-depth: 2
-          token: ${{ secrets.FLOWZONE_TOKEN }}
-
-      - name: Assert that versioned source is up-to-date
-        if: ${{ steps.versionbot_checkout.outcome == 'success' }}
-        run: git merge-base --is-ancestor ${{ github.event.pull_request.head.sha }} HEAD
+      # - name: Checkout versioned source
+      #   id: versionbot_checkout
+      #   continue-on-error: true
+      #   uses: actions/checkout@v3
+      #   with:
+      #     ref: versionbot/pr/${{ github.event.pull_request.number }}
+      #     fetch-depth: 2
+      #     token: ${{ secrets.FLOWZONE_TOKEN }}
 
       # Fallback to unversioned source if versionbot branch does not exist
       - name: Checkout original source
-        if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
+        # if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
         uses: actions/checkout@v3
         with:
-          fetch-depth: 1
           token: ${{ secrets.FLOWZONE_TOKEN }}
 
+      # - name: Assert that versioned source is up-to-date
+      #   # if: ${{ steps.versionbot_checkout.outcome == 'success' }}
+      #   run: git merge-base --is-ancestor ${{ github.event.pull_request.head.sha }} HEAD
+
       # Run balena-versionist in Github actions when the versionbot branch could not be checked out
       - name: Version original source
-        if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
+        # if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
         run: |
           npm install -g balena-versionist versionist
           balena-versionist
+          git status
+          VERSION="$(head -n1 VERSION || jq -r '.version' package.json)"
+          git add --all
+          git -c "user.name=Versionist" -c "user.email=versionist@balena.io" commit --allow-empty -m "${VERSION}"
```

Comment:
Can we add an input to skip this if versioning is done manually?

Reply:
I added the input variable `skip_versioning` to skip the versioning step if needed.